lang/cs/Org.Apache.REEF.Tang/Implementations/Configuration/ConfigurationImpl.cs

```diff
@@ -15,9 +15,9 @@
 // specific language governing permissions and limitations
 // under the License.
 
-using System.Collections.Generic;
 using Org.Apache.REEF.Tang.Interface;
 using Org.Apache.REEF.Tang.Types;
+using System.Collections.Generic;
 
 namespace Org.Apache.REEF.Tang.Implementations.Configuration
```

Comment:
Changes in this file seem superficial only. Maybe revert?

Reply:
It was sorting the using - but there are no additional changes so I dont think its justified. I will revert.

---
In reply to: [223779942](https://github.com/apache/reef/pull/1479#discussion_r223779942) [](ancestors = 223779942)